### ISSUE TITLE
GG MAX - Please hurry and FIX This bug with pressing PLAY GAME from the hub

### ISSUE THREAD

**Issue by Tum1370** [TheGameCreators/GameGuruRepo#2868]
This bug is really starting to annoy me now.
Its not just the fact the the demo games have no gun, Its now affecting TESTING ALL levels of your own game.
Also, how can only a few of us be having this issue ?????

https://github.com/TheGameCreators/GameGuruRepo/issues/2844
https://github.com/TheGameCreators/GameGuruRepo/issues/2405

This is driving me crazy. I have even thought about reinstalling Windows, But that would waste a load of time, Mess up all my other programs like Vortex and game mods i have installed, And would probably not fix the problem.

**Comment by Necrym59** [comment]
You have already said some things were fixed after a reinstall in discord maybe there is part of your answer, perhaps you should have a careful look at your system. I have tried myself to repoduce every thing you have had problems with and had no errors on 3 comps of varying specs and only one graphic glitch due to an older driver.

**Comment by Tum1370** [comment]
Everything is updated on this new pc. 
I have not said I fixed anything on discord, all I did there was make several backups of my game projects folder so I was lucky and could roll back after that last bug. 
This bug is stopping me test ALL levels, which I cannot fix by rolling back a saved folder. 

All drivers are up to date. These are already things I have tried. 

This is not a bug that I can fix by updating anything my end. 

Did you try this on a PC with a new 12900k cpu ?

That is what I think maybe the problem.

**Comment by Necrym59** [comment]
I have a 12900k cpu 12th Gen i9 and dont have any problems there its strange if you reinstall go through and delete everything max has perhaps left behind in registry or temp folders

**Comment by Tum1370** [comment]
I have checked all hidden app data and nothing for Max there. 
There are registry settings but I don’t know what or where they all are. 
That’s what the deva will know. 

All I know is currently I have done everything I possibly can and no fix.

**Comment by Necrym59** [comment]
run regedit and do a search for MAX should find somthing in registry

**Comment by Tum1370** [comment]
I already have. Like I said there are settings but I don’t understand what they mean. 
And they are scattered in several places.

**Comment by Necrym59** [comment]
Yes im looking now its a bit spread out and messy

**Comment by Tum1370** [comment]
Yes. Well my last report on this was on one of the links in the first post. 
It was trying to Play ALL LEVELS from the storyboard. 
But the software gave a runtime error when trying to load next level. 
So am hoping that will pinpoint part of this problem.

**Comment by plemsoft** [comment]
@Tum1370 Please try this and let me know if it fix all your issues:

In the MAX install folder , locate this file "setup.ini" , edit the file in notepad.

Locate this line:
"vrmode=3"
And change it to
"vrmode=0"

If it do fix your issues, please make sure to close down all your GitHub issues. Thanks for your help :)

**Comment by Tum1370** [comment]
@plemsoft 
Ok thanks. I’ll will try this as soon as I am home from work. I have seen that line when I’ve looked through the setup file. 

Thing is though, why would this line be affecting just mine and a few other people pcs ?

because I guess everyone else has that vrmode set to 3 as well, without issues ?

**Comment by plemsoft** [comment]
@Tum1370 I think you have OpenVR steam or Windows Mixed Reality active on your system, VR is not yet ready :)

**Comment by Tum1370** [comment]
@plemsoft 
Ah ok. That’s good to hear. 
yes I definitely have steamvr and openvr active as I only ever play vr games now. 

Looking forward to check this when home now. Thanks

**Comment by Tum1370** [comment]
@plemsoft @LeeBamberTGC 

Thank you so much for figuring that bug out for me Preben.
After changing the "vrmode=3" to "vrmode=0" i now have the guns visible in all the demo games that i have just tested.
I also tested the one demo game that has 2 levels, by using the TEST ALL LEVELS, And this time it would load in to the 2nd level after completing the first level.
The only BUG that i see was when i tried TESTING ALL LEVELS from Storyboard, I did try the "INVULNERABLE STANDALONE" but this was not INVULNERABLE. I still had health, and died. 

But yes, this has fixed all these following bugs that i was suffering with,

The MISSING GUN from when you press PLAY GAME from the hub on Demo games, and on my OWN games.
It also fixed the bug when playing using "STANDALONE TEST ALL LEVELS" from the storyboard.

Thank you very much.
